### ISSUE TITLE
No Pro membership but still able to use the composition feature.

### ISSUE THREAD

**Issue by dcyaner** [tokens-studio/figma-plugin#1572]
Is it possible to use the composition feature by manipulating the json even if I don't have a Token Studio PRO membership? Is this something that is done deliberately?

I have no ill intent, I am just using Token Studio to practice creating design tokens before purchasing a Pro membership.

**Comment by six7** [comment]
Yeah, our pro tier works so that Free users can use it, just not make edits.  We want to enable other designers to benefit from those features, and not lock those out. Otherwise working in a team would be quite difficult.

You could also do that with json as you suggest.

**Comment by dcyaner** [comment]
According to what you said, I realized that I can also make edits. I was just using it to gain experience, and if I need it, I will switch to the pro version when I learn.

![image](https://user-images.githubusercontent.com/74204161/215453552-ac6013f6-70b3-47e3-8926-bfe5d214e892.png)
